app/src/main/res/layout/activity_select_calendar.xml

```diff
@@ -16,7 +16,7 @@
         app:layout_constraintStart_toStartOf="parent"/>
 
     <Button
-        android:id="@+id/gotochooseaccount"
+        android:id="@+id/gotosharingoptions"
```

Comment:
for resource naming, check:
https://github.com/umesh0492/android-guidelines/blob/master/project_and_code_guidelines.md
**2.3.2 Resources naming**